resources/views/user/component/information_detail.blade.php

```diff
@@ -29,22 +29,49 @@
             </section>
             <section class="change-email">
                 <section>
-                    <form action="">
+                    <form action="{{ route('update', auth()->user()->id) }}" method="POST" id="update-information-form">
+                        @method('PATCH')
+                        @csrf
+                        <input type="text" class="d-none" value="{{ config('validate.update_information') }}" name="validate_rule">
                         <div class="input-container">
                             <label for="">@lang('label.full_name')</label>
                             <div>
-                                <input type="email" maxlength="50" placeholder="@lang('label.enter_full_name')" value="{{ $user->name }}">
+                                <input type="text" placeholder="@lang('label.enter_full_name')" value="{{ $user->name }}" name="name" class="@error('name') error-focus @enderror">
+                                @error('name')
+                                    <p class="error-message">{{ $message }}</p>
+                                @enderror
                             </div>
                         </div>
                         <div class="input-container">
                             <label for="">@lang('label.date_of_birth')</label>
                             <div>
-                                <input type="text" maxlength="50" placeholder="@lang('label.enter_date')" value="{{ $user->date_of_birth }}">
+                                <input type="text" maxlength="50" placeholder="@lang('label.enter_date')" value="{{ $user->date_of_birth }}" name="date_of_birth" class="@error('date_of_birth') error-focus @enderror">
+                                @error('date_of_birth')
+                                    <p class="error-message">{{ $message }}</p>
+                                @enderror
+                            </div>
+                        </div>
+                        <div class="input-container">
+                            <label for="">@lang('label.phone')</label>
+                            <div>
+                                <input type="text" maxlength="15" placeholder="@lang('label.phone_number')" value="{{ $user->phone }}" name="phone" class="@error('phone') error-focus @enderror">
```

Comment:
style em viết ra file css nhé. anh thấy e viết rất nhiều style vào view. fix hết toàn bộ e nhé

Reply:
em có check lại thi toàn bộ style em cho hết vào css rồi mà anh, có phải ý anh bảo là cái maxlength không ạ?